### ISSUE TITLE
Owner content item filter

### ISSUE THREAD

**Issue by agriffard** [OrchardCMS/OrchardCore#14343]
Is there an `Owner` filter in the content items search?

I don't seem to find one in the available filters of the modal with syntax examples.

**Comment by MikeAlhayek** [comment]
I don't recall having one. If you have `List Own Contents` then you can only see your own items. But, I think what you are looking for does not exists in OC.

**Comment by agriffard** [comment]
That can be useful when you start creating contents per user and then want to be able to filter them.

An actual bootstrap-select filter would be also interesting but it would be the first one whose number elements could grow (if more and more users are added) and cause performance pbs.

**Comment by MikeAlhayek** [comment]
Yes. If you would to implement such filer, you may need to use what we use in the UserFieldPicker.
